public/angular/js/controllers/media/settings.js

```diff
@@ -19,6 +20,10 @@ angular.module('Aggie')
       var modalInstance = modal.create(media, $scope.media[media]);
     };
 
+    $scope.inHttp = false;
+    if ($window.location.protocol === 'http:') {
+      $scope.inHttp = true;
+    }
```

Comment:
You can simplify these 4 lines to `$scope.inHttp = ($window.location.protocol === 'http:')`